modules/index-service/src/main/java/org/opencastproject/index/service/impl/index/event/EventUtils.java

```diff
@@ -44,175 +45,190 @@
-        MetadataField<?> startDate = metadata.getOutputFields().get("startDate");
-        metadata.removeField(startDate);
-        MetadataField<String> newStartDate = MetadataUtils.copyMetadataField(startDate);
-        SimpleDateFormat sdf = new SimpleDateFormat(startDate.getPattern().get());
-        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
-        newStartDate.setValue(sdf.format(startDateTime));
-        metadata.addField(newStartDate);
-      }
-    }
-
-    if (event.getDuration() != null) {
-      MetadataField<?> duration = metadata.getOutputFields().get("duration");
-      metadata.removeField(duration);
-      MetadataField<String> newDuration = MetadataUtils.copyMetadataField(duration);
-      newDuration.setValue(event.getDuration().toString());
-      metadata.addField(newDuration);
-    }
-
-    if (metadata.getOutputFields().containsKey("location")) {
-      MetadataField<?> agent = metadata.getOutputFields().get("location");
-      metadata.removeField(agent);
-      MetadataField<String> newAgent = MetadataUtils.copyMetadataField(agent);
-      newAgent.setValue(event.getLocation());
-      metadata.addField(newAgent);
-    }
-
-    if (metadata.getOutputFields().containsKey(DublinCore.PROPERTY_SOURCE.getLocalName())) {
-      MetadataField<?> source = metadata.getOutputFields().get(DublinCore.PROPERTY_SOURCE.getLocalName());
-      metadata.removeField(source);
-      MetadataField<String> newSource = MetadataUtils.copyMetadataField(source);
-      newSource.setValue(event.getSource());
-      metadata.addField(newSource);
-    }
-
-    if (metadata.getOutputFields().containsKey(DublinCore.PROPERTY_CREATED.getLocalName())) {
-      String createdDate = event.getCreated();
-      if (StringUtils.isNotBlank(createdDate)) {
-        MetadataField<?> created = metadata.getOutputFields().get(DublinCore.PROPERTY_CREATED.getLocalName());
-        metadata.removeField(created);
-        MetadataField<Date> newCreated = MetadataUtils.copyMetadataField(created);
-        newCreated.setValue(new Date(DateTimeSupport.fromUTC(createdDate)));
-        metadata.addField(newCreated);
-      }
-    }
-
-    if (metadata.getOutputFields().containsKey(DublinCore.PROPERTY_IDENTIFIER.getLocalName())) {
-      MetadataField<?> uid = metadata.getOutputFields().get(DublinCore.PROPERTY_IDENTIFIER.getLocalName());
-      metadata.removeField(uid);
-      MetadataField<String> newUID = MetadataUtils.copyMetadataField(uid);
-      newUID.setValue(event.getIdentifier());
-      metadata.addField(newUID);
-    }
-
-    return metadata;
-  }
-
-  /**
-   * A filter to remove all internal channel publications.
-   */
-  public static final Fn<Publication, Boolean> internalChannelFilter = new Fn<Publication, Boolean>() {
-    @Override
-    public Boolean apply(Publication a) {
-      if (InternalPublicationChannel.CHANNEL_ID.equals(a.getChannel()))
-        return false;
-      return true;
-    }
-  };
+    /** The logging facility */
```

Comment:
Opencast uses 2 spaces for indentation in Java code (see https://docs.opencast.org/develop/developer/code-style/). Please revert this back to 2 spaces.